src/components/ToyForm.js

```diff
@@ -20,15 +20,17 @@ const ToyForm = ({ toy }) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    dispatch(createToy(form, toy));
+    const { imageUrl, ...rest } = form;
+    dispatch(createToy({ image_url: imageUrl, ...rest }, toy));
```

Comment:
This is looking better, but when handing things like tweaking field names for sending via an API call, think about where the responsibility lies.  Is it the components responsibility to care what the data looks like when making the API call?  Probably not, it shouldn't care about the API at all.  It feels like responsibility should be where the API call is being made-- the `createToy`action.